index.md

```diff
@@ -202,32 +202,32 @@ description: Vanilla OS is an Immutable Linux-based distribution which aims to p
             <div class="media-carousel">
                 <div class="media-carousel-item">
                     <a href="https://www.zdnet.com/article/vanilla-os-offers-a-new-take-on-security-for-the-linux-desktop/" target="_blank" rel="noopener">
-                        <img src="/assets/images/logos/media/zdnet.svg" alt="ZDNet" />
+                        <img src="/assets/images/logos/media/zdnet.svg" alt="ZDNet logo." />
                         <p>If you're looking for a new operating system that takes security seriously… Vanilla OS might be just the ticket.</p>
                     </a>
                 </div>
                 <div class="media-carousel-item">
                     <a href="https://www.theregister.com/2023/01/03/vanilla_os_2210/" target="_blank" rel="noopener">
-                        <img src="/assets/images/logos/media/the_register.svg" alt="The Register" />
+                        <img src="/assets/images/logos/media/the_register.svg" alt="The Register logo." />
                         <p>Vanilla OS is going for bigger, bolder improvement to the Linux desktop… it could prove transformative to the Linux industry… </p>
                     </a>
                 </div>
                 <div class="media-carousel-item">
                     <a href="https://www.techrepublic.com/article/vanilla-os-linux-desktop/" target="_blank" rel="noopener">
-                        <img src="/assets/images/logos/media/techrepublic-logo.svg" alt="ABRoot" />
+                        <img src="/assets/images/logos/media/techrepublic-logo.svg" alt="Tech Republic logo." />
```

Comment:
```suggestion
                        <img src="/assets/images/logos/media/techrepublic-logo.svg" alt="Tech Republic: "This will be a project you’ll want to keep a close watch on... I think Vanilla OS could be a real game-changer."" />
```